lib/oneview-sdk/resource/api200/uplink_set.rb

```diff
@@ -78,6 +85,13 @@ def add_fcoenetwork(network)
         raise IncompleteResource, 'Must set network uri attribute' unless uri
         @data['fcoeNetworkUris'].push(uri)
       end
+
+      # Gets the unassigned uplink ports
+      def get_unassigned_ports
```

Comment:
Ok, makes sense to implement it if the UI also uses it. 
Still needs expected ``@return``/``@params`` comments for documentation.